doc/help/commands.asciidoc

```diff
@@ -122,6 +122,8 @@ Save the current page as a bookmark, or a specific url.
 
 If no url and title are provided, then save the current page as a bookmark. If a url and title have been provided, then save the given url as a bookmark with the provided title.
 
+You can view all saved bookmarks on the link:qute://bookmarks[bookmarks page].
+
```

Comment:
This doc is autogenerated from the docstrings of the respective `@cmdutils.register` decorated command functions - please change those and run `scripts/dev/src2asciidoc.py`.